tests/functional_small/solvers_nonlinear/standard_gauss_newton_normal_equations/gn_w_basic_backtrack_eigen.cc

```diff
@@ -0,0 +1,122 @@
+//
+#include "pressio/solvers.hpp"
+struct ValidSystem 
+{
```

Comment:
can you move this problem class into the problems directory?

Reply:
done